src/Agent.Plugins/BuildDrop/BuildServer.cs

```diff
@@ -42,5 +44,15 @@ public BuildServer(VssConnection connection)
 
             return await _buildHttpClient.CreateArtifactAsync(artifact, projectId, buildId, cancellationToken: cancellationToken);
         }
+
+        // Get named artifact from a build
+        public async Task<BuildArtifact> GetArtifact(
+            Guid projectId,
+            int buildId,
+            string name,
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return await _buildHttpClient.GetArtifactAsync(projectId, buildId, name, userState: null, cancellationToken: cancellationToken);
```

Comment:
i think you can remove `userState: null`

Reply:
removed